README.md

```diff
@@ -2,9 +2,9 @@
 Hardware dimensions usable in mechanical design.
 
 ##Function
-Dimensions are stored in Open Document Spreadsheets (ods). CSV and JSON files generated by the build.sh script are available. LibreOffice is used to generate the CSV files.
+Dimensions are stored in  JSON files generated BowlerStudio. 
```

Comment:
BowlerStudio is generating JSON from GSON and an internal datatype tuned to match the schema. Could we do a JSON ->CSV instead? JSON tends to be the "universal" format these days, and it seem bad to have more then one copy of "the data". Only one "copy" of the data should be the "true" data, and the other generated. i guess it comes down to the purpose, machine or human readable?